pkg/network/tracer/tracer.go

```diff
@@ -182,6 +185,19 @@ func newTracer(cfg *config.Config, telemetryComponent telemetryComponent.Compone
 	tr.reverseDNS = newReverseDNS(cfg, telemetryComponent)
 	tr.usmMonitor = newUSMMonitor(cfg, tr.ebpfTracer)
 
+	// Setup the connection_protocol map cleaner if protocol classification is enabled and USM is disabled
+	if cfg.ProtocolClassificationEnabled && !usmconfig.IsUSMSupportedAndEnabled(cfg) {
+		connectionProtocolMap := tr.ebpfTracer.GetMap(probes.ConnectionProtocolMap)
+		if connectionProtocolMap == nil {
+			log.Warn("could not get connection_protocol map, will not be able to expire connection_protocol data")
```

Comment:
you are logging twice, once in GetMap and once here

Reply:
I'll remove this log thanks, we want the underlying log from the ebpf tracer